### DISCUSSION TITLE
pikpak上传 --2 validation error(s) found.

### DISCUSSION THREAD

**Discussion by ika166** [AlistGo/alist#1451] [Invalid question]
2 validation error(s) found. - minimum field size of 1, CreateMultipartUploadInput.Bucket. - minimum field size of 1, CreateMultipartUploadInput.Key.怎么解决

**Comment by ika166** [comment]
大小文件都不行

**Comment by l3ony2k** [comment]
我也遇到了这个问题，我试了用 WebDav 也是提示上传失败。
